KsApi/models/Backing.swift

```diff
@@ -36,16 +36,17 @@ public func == (lhs: Backing, rhs: Backing) -> Bool {
 extension Backing: Decodable {
   public static func decode(_ json: JSON) -> Decoded<Backing> {
     let create = curry(Backing.init)
-    let tmp = create
+    let tmp1 = create
```

Comment:
argo was of course the cause of quite a few slow functions, so i did a lot of work on the slowest ones...